pkg/seeds/seeds.go

```diff
@@ -75,7 +75,7 @@ func SeedRepositoryConfigurations(db *gorm.DB, size int, options SeedOptions) er
 	return nil
 }
 
-func randomLastRead() (lastReadTime *time.Time, lastReadError *string) {
+func randomLastRead() (lastIntrospectionTime *time.Time, lastIntrospectionError *string) {
```

Comment:
:question: should this return the 5 values? (LastIntrospect* and Status)

Reply:
Yes, I will update this next